.github/workflows/ci.yml

```diff
@@ -4,10 +4,11 @@ on:
   push:
     branches:
       - master
+      - main
```

Comment:
Issue: This change isn't necessary

Reply:
Reverted it. It was to test it on my branch.